dcat/config.js

```diff
@@ -2,12 +2,12 @@ var respecConfig = {
     doJsonLd: true,
 //	preProcess: [dfn_index],
 //    subtitle: "Version 2",
-    specStatus: "ED",
+  specStatus: "PR",
```

Comment:
```suggestion
  specStatus: "ED",
```
Since this will reflect automatically on the "live" version of the document on github pages, this should not be changed here.
Once the rest of this PR is merged, I will generate a publication snapshot with the correct option.